src/Features/VisualBasic/Portable/QuickInfo/VisualBasicQuickInfoService.vb

```diff
@@ -0,0 +1,27 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.QuickInfo
+Imports Microsoft.CodeAnalysis.Host.Mef
+Imports Microsoft.CodeAnalysis.Host
+
+Namespace Microsoft.CodeAnalysis.CSharp.QuickInfo
```

Comment:
Yep.  That's funny.  Maybe someone will use it to prove that VB is being slowly turned into C#.

---
In reply to: [77707144](https://github.com/dotnet/roslyn/pull/13623#discussion_r77707144) [](ancestors = 77707144)